mock-wallet/Cargo.toml

```diff
@@ -6,3 +6,10 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+libsecp256k1 = "0.7.1"
```

Comment:
this crate is deprecated now?

can still first go to develop, and we'll delete it later separately;

only skimming review

Reply:
Yeah, let's remove it already